### ISSUE TITLE
Documentation: Credits page needs to be updated

### ISSUE THREAD

**Issue by leahcimto** [theforumhelpers/theforumhelpers.github.io#248]
**What does this affect?**
Credits page

**Changes/Additions**
The order of users needs to be changed.

gosoccerboy5 should be moved after leahcimto, BarelySmooth should be added after NilsTheBest, FunctionalMetatable should be after BarelySmooth, Shluffy should be after FunctionalMetatable, and TheColaber should be added after Nambaseking01.

The grid should look like this:
<img width="1031" alt="Screen Shot 2021-06-16 at 5 42 59 PM" src="https://user-images.githubusercontent.com/68165163/122298076-4c139700-ceca-11eb-95c6-c5d0b67f9224.png">

The contributors variable should be updated to the following:
```js
var contributors = ["Accio1", "leahcimto", "gosoccerboy5", "NilsTheBest", "BarelySmooth", "FunctionalMetatable", "Shluffy", "Nambaseking01", "TheColaber", "shefwerld", "ShadowOfTheFuture", "bremea", "awesome-llama", "CodeGuy92", "jeffalo", "MTM828", "The64thCucumber"]
```

I have a branch for this and a PR should be up soon.

**Comment by Steve0Greatness** [comment]
Why?

**Comment by gosoccerboy5** [comment]
I don't think I deserve to be the 3rd

**Comment by leahcimto** [comment]
> Why?

@Steve0Greatness This is how the credits page is set up. Users are placed in order of contributions they make, and users with no commits are placed after them in alphabetical order.

**Comment by Accio1** [comment]
Closed in #249
